### DISCUSSION TITLE
Can I contribute to documentation? (discordpy.readthedocs.io)

### DISCUSSION THREAD

**Discussion by eamonburns** [Rapptz/discord.py#9436] [General] [answered]
I have found a small error in the [documentation](https://discordpy.readthedocs.io/), I think it is a very small thing that I could fix myself. Is there a GitHub repo that I can contribute to that the documentation is from?

**Comment by Vexs** [comment]
The documentation is built from the source- so, this is the repo.

**Comment by Gobot1234** [comment, marked as answer]
The documentation is stored either in the docstrings for the relevant method or the docs directory in this repo itself. If you need any help contributing people in the server will be more than happy to help https://discord.gg/r3sSKJJ
